gcp-py-instance-nginx/README.md

```diff
@@ -0,0 +1,237 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# Pulumi Nginx Server in a instance (GCP)
+
+Starting point for building the Pulumi nginx server sample in Google Cloud Platform.
+
+## Running the App
+
+1.  Create a new stack:
+
+    ```
+    $ pulumi stack init gcp-instance-nginx
+    ```
+
+2.  Configure the project:
+
+    ```
+    $ export GOOGLE_PROJECT=cncf-230209; export GOOGLE_REGION=asia-east1; export GOOGLE_ZONE=asia-east1-a;
+    $ export GOOGLE_CREDENTIALS=YOURGCPCREDENTIALS
+    ```
+
+3.  Run `pulumi up` to preview and deploy changes:
```

Comment:
Before we run `pulumi up` I think we need to configure the Python project:

```
virtualenv -p python3 venv
source venv/bin/activate
pip install pulumi_gcp
```

Can you add this between steps 2 and 3?

Reply:
Done. Thanks for the suggestion:)